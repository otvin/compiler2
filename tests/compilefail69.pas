program compilefail69(output);
{typo in for statement}


var i:integer;


begin
	
	for i is 17 to 19 do 
		writeln(i);

end.
