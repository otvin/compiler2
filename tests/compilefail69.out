Expected ':=' but saw 'is' in file:tests/compilefail69.pas line:10    column:11 
